pennylane/templates/embeddings/iqp.py

```diff
@@ -166,20 +187,19 @@ def circuit(features=None, pattern=None):
 
             assert np.allclose(res1, res2)
 
-        **Repeating the embedding**
+        **Non-consecutive wires**
 
-        The embedding can be repeated by specifying the ``n_repeats`` argument:
+        In principle, the user can also pass a non-consecutive wire list to the template.
+        For single qubit gates, the i'th feature is applied to the i'th wire index (which may not be the i'th wire).
+        For the entanglers, the product of i'th and j'th features is applied to the wire indices at the i'th and j'th
+        position in ``wires``.
 
-        .. code-block:: python
+        For example, for ``wires=[2, 0, 1]`` the ``RZ` block applies the first feature to wire 2,
+        the second feature to wire 0, and the third feature to wire 1.
 
-            @qml.qnode(dev)
-            def circuit(features=None):
-                IQPEmbedding(features=features, wires=range(3), n_repeats=4)
-                return [qml.expval(qml.PauliZ(w)) for w in range(3)]
-
-            circuit(features=[1., 2., 3.])
-
-        Every repetition uses exactly the same quantum circuit.
+        Likewise, using the default pattern, the entangler block applies the product of the first and second
+        feature to the wire pair ``[2, 0]``, the product of the second and third feature to ``[2, 1]`` and so
```

Comment:
```suggestion
        feature to the wire pair ``[2, 0]``, the product of the second and third feature to ``[0, 1]`` and so
```
I'm not sure though

Reply:
I think it was correct. I know it's a bit convoluted - but the second wire pair in the default pattern is the first and third wire ``[2,1]``...